### PR TITLE
Add new Foxeer F722 V4 MPU6000 Target

### PR DESCRIPTION
**Pull request by Johnsnow-foxeer**
Add new target base on MPU6000 gyro, barometer is on board.

### CODE REVIEW

File: configs/default/FOXE-FOXEERF7V4M6000.config [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# Betaflight / STM32F7X2 (S7X2) 4.0.0 Apr  4 2019 / 00:54:12 (e44f75b4a) MSP API: 1.41
+
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACC_SPI_MPU6000
+#define USE_BARO_DPS310
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+
+board_name FOXEERF7V4M6000
```
Comment (haslinghuis) [inline configs/default/FOXE-FOXEERF7V4M6000.config:9]: Should we just call it V5 as this looks a lot like V4 + BARO
Or we could add the BARO to V4

Comment (Johnsnow-foxeer) [reply configs/default/FOXE-FOXEERF7V4M6000.config:9]: V4, please, because the board is printed V4 already

Comment (haslinghuis) [reply configs/default/FOXE-FOXEERF7V4M6000.config:9]: Okay NP... just add the defines to the V4. No need for a new target :) - will update for 4.5 too.

Comment (Johnsnow-foxeer) [reply configs/default/FOXE-FOXEERF7V4M6000.config:9]: Added the defines, tested, the barometer doesn't work, so add a new target

Comment (haslinghuis) [reply configs/default/FOXE-FOXEERF7V4M6000.config:9]: Nah, we are fixing that.

**Review by sugaarK — APPROVED** [review]

**Comment by haslinghuis** [comment]
@sugaarK we will be updating V4 with the MPU6000 instead of creating a new one as it's already in production as V4.

File: configs/default/FOXE-FOXEERF7V4M6000.config (hunk 1 above)
Comment (haslinghuis) [reply configs/default/FOXE-FOXEERF7V4M6000.config:9]: @Johnsnow-foxeer be sure to enable expert mode on the firmware flasher tab (it's hidden beneath show release candidates. Then add in custom defines: `ACC_SPI_MPU6000 GYRO_SPI_MPU6000` and flash 4.4.1 (or 4.5)

**Comment by sugaarK** [comment]
I did that already I'll close this

**Pull request closed without merging**